modules/apps/frontend-taglib/frontend-taglib-clay-sample-web/src/main/java/com/liferay/frontend/taglib/clay/sample/web/internal/display/context/SearchPaginatorDisplayContext.java

```diff
@@ -0,0 +1,85 @@
+/**
```

Comment:
This one has to be moved to the other sample as well

Reply:
Done!